src/model/bpmn/edge/SequenceFlowKind.ts

```diff
@@ -16,4 +16,5 @@
 export enum SequenceFlowKind {
   NORMAL = 'normal',
   DEFAULT = 'default',
+  CONDITIONAL = 'conditional',
```

Comment:
:warning: we have forgotten to update the documentation when we introduced the `SequenceFlowKind`
we should have document what to add when supporting new flow node types. Do we have other types to support? if so we should update the doc, otherwise, useless :smiley_cat:

Reply:
I updated the internal model to ad it. I don't know if we need to add this in another place.

Reply:
What I have in mind in the `architecture/bpmn-support-how-to.adoc` documentation